### ISSUE TITLE
Integrate SauceLabs using CodeceptJS and Playwright Helper

### ISSUE THREAD

**Issue by jpaxtonrs** [Codeception/CodeceptJS#3482]
I am trying to integrate my `CodeceptJS` tests using the `Playwright` Helper with `SauceLabs` so I can have my tests run on various devices in the Cloud.

I see documentation on how to do it using the `Appium` and `WebDriver` Helpers, but, nothing about `Playwright` Helper.

I tried using the `Node.js` wrapper for `saucectl` and it only supports the native `Playwright` framework, not the Helper under the hood for `CodeceptJS`. 

Am I missing some documentation that can lead me into the right direction allowing me to run my tests in `SauceLabs`?
If this is not achievable at the moment, are there plans in the future to complete this integration like you all have for `Appium`?

Thanks!

**Comment by kobenguyent** [comment]
https://docs.saucelabs.com/web-apps/automated-testing/playwright/

As I read here, your Playwright tests are running against browsers yet the devices by saucelabs. Or something I understand wrongly 🤔

**Comment by jpaxtonrs** [comment]
Technically, they are not Playwright tests. They are CodeceptJS tests that use the Playwright Helper - https://codecept.io/playwright/

When I use the Playwright solution with `saucectl` , I receive failures as the solution is based on the native Playwrtight framework (not CodeceptJS with Playwright).

The idea is to have my CodeceptJS tests run in Saucelabs against various real device and browser combinations in the Cloud.

**Comment by kobenguyent** [comment]
Yeah @jpaxtonrs understood your use case. Seems like the integration should be implemented cause currently this wouldn't work out of the box for Playwright helper of Codeceptjs.

**Comment by jpaxtonrs** [comment]
Agreed.

With that said, could I request for the Implementation of SauceLabs integration with CodeceptJS + Playwright Helper (like Codecept has for Appium) ?

**Comment by kobenguyent** [comment]
Checked with SL. Need to wait for them then. 

![image](https://user-images.githubusercontent.com/7845001/199305080-ed5a769e-582a-483b-947f-bfa51d224969.jpeg)

**Comment by jpaxtonrs** [comment]
Awesome, thanks so much! Please, keep me updated on the status of this.
